### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by jcalonso**
Add a simple Dockerfile to allow people running JsonVisio using Docker.
Later this Dockerfile could be used in a GitHub action to provide and official Docker image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,15 @@ JSON Visio is data visualization tool for your json data which seamlessly illust
   <img width="800" src="https://github.com/AykutSarac/jsonvisio.com/blob/main/public/preview_2.png" alt="preview 1" />
   </div>
 
+## Docker
+
+A Docker file is provided in the root of the repo.
+If you want to run JsonVision locally:
+
+* Build Docker image with `docker build -t jsonvisio.`
```
Comment (mg6) [inline README.md:62]:
```suggestion
* Build Docker image with `docker build -t jsonvisio .`
```

**Review by mg6 — COMMENTED** [review]
Works nicely with the fix, thanks!

**Comment by jcalonso** [comment]
Thanks for the review 🙂
Added your fix 👍🏼

**Review by mg6 — APPROVED** [review]
Looks good 🚀

**Comment by AykutSarac** [comment]
Thanks for your contribution! 🚀

**Pull request merged**